content/en/tracing/profiler/enabling/ddprof.md

```diff
@@ -14,34 +16,45 @@ further_reading:
 ---
 
 <div class="alert alert-warning">
-The Datadog Profiler for Linux is in public beta. Datadog recommends evaluating the profiler in a non-sensitive environment before deploying in production.
+The compiled languages profiler, <code>ddprof</code> is in public beta. Datadog recommends evaluating the profiler in a non-sensitive environment before deploying in production.
 </div>
 
-The Datadog Profiler for Linux lets you collect profile data for applications running on Linux. It is ideally suited for applications written in compiled languages, such as C, C++, or Rust. It can also profile services such as nginx, Redis, or Postgres, as it does not require access to source code.
+The Datadog Profiler for compiled languages (`ddprof`) uses OS level APIs to collect profiling data. It is ideally suited for applications written in compiled languages, such as C, C++, or Rust.
 
 ## Requirements
 
-The Datadog Profiler for Linux is compatible with Linux v4.17+ on `amd64` or `arm64` processors. It is not supported on serverless platforms, such as AWS Lambda, or on operating systems other than Linux, such as Windows, BSD, and MacOS.
+Supported operating systems
+: Linux kernel v4.17+, libc 2.23+
 
-## Standalone installation
+Supported architecture
+: `amd64` or `arm64` processors
 
-The profiler can be used either as a standalone executable or as a library. Skip to [library installation instructions](#library-installation) if you want to use it as a library. Otherwise, to begin profiling applications with the standalone executable:
+Serverless
+: `ddprof` is not supported on serverless platforms, such as AWS Lambda.
 
-1. Configure the `perf_event_paranoid` kernel setting to be at most 2. On many distributions, you can check this parameter by running `cat /proc/sys/kernel/perf_event_paranoid` or `sysctl kernel.perf_event_paranoid`. You can set this value until the next reboot by running `echo 2 | sudo tee /proc/sys/kernel/perf_event_paranoid` and set it on every system startup with `sudo sysctl -w kernel.perf_event_paranoid=2`. These commands can't usually be run from a container, as the `perf_event_paranoid` setting is an operating system parameter. These commands may not work for all configurations. For alternatives, see [Troubleshooting][1].
+OS Settings
+: `perf_event_paranoid` kernel setting is 2 or less (see [Troubleshooting][1])
 
-2. Ensure you are running the Datadog Agent version [7.20.2][2]+ or [6.20.2][3]+.
+Debugging information
+: Symbols should be available
```

Comment:
I think we should be more specific by saying something like, "ddprof will not be able to provide human-readable function names if the symbol table is stripped."  This communicates two things

* The user may not be aware that "symbols should be available" means "the ELF symbol table must be present in the main binary."  However, they may be aware that they "strip" their executables to save room (this is becoming more and more common).  Moreover, I think we should speak about the thing (the symbol table) rather than the information it provides (symbol names).
* ddprof kinda works if the symbol table is missing, it's just that the resulting flamegraphs are annotated with execution addresses rather than symbol names.  Pointing out that the user won't see human-readable symbol names also allows them to self-serve in some of the diagnostics.

WDYT?

Reply:
I added that as an additional clarification on the existing sentence. Let me know what you think.

Reply:
Thanks, this is indeed more and more common. It was worth adding a sentence.